### PR TITLE
hierarchical on demand subworkflow loading

### PR DESCRIPTION
**Pull request by soson**

### CODE REVIEW

File: packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-table/executed-workflow-hierarchical-table/executed-subworkflow-table.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  isLoading: boolean;
+  subWorkflows: ExecutedSubworkflowTask[];
+};
+type Props = {
+  workflowId: string;
+  subWorkflows: Map<string, ExecutedSubWorkflows>;
+};
+const ExecutedSubWorkflowTable = ({ workflowId, subWorkflows }: Props) => {
+  const workflows = subWorkflows.get(workflowId);
+  console.log(workflows);
```
Comment (MarcoMruz) [inline packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-table/executed-workflow-hierarchical-table/executed-subworkflow-table.tsx:17]: still one log :)

File: packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx [hunk 2]
```diff
@@ -145,6 +145,8 @@ const ExecutedWorkflowList = () => {
     setPagination((prev) => ({ ...prev, page: pageNumber }));
   };
 
+  console.log(pagination);
```
Comment (MarcoMruz) [inline packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx:148]: another one

**Review by MarcoMruz — DISMISSED** [review]

File: packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-table/executed-workflow-hierarchical-table/executed-subworkflow-table.tsx (hunk 1 above)
Comment (soson) [reply packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-table/executed-workflow-hierarchical-table/executed-subworkflow-table.tsx]: removed

File: packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx (hunk 2 above)
Comment (soson) [reply packages/frinx-workflow-ui/src/pages/workflow-list/executed-workflow-list/executed-workflow-list.tsx]: removed

**Review by MarcoMruz — APPROVED** [review]

**Pull request merged**